dashboard/db/migrate/20160213001925_add_submitted_to_user_levels.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubmittedToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :submitted, :boolean
+  end
```

Comment:
The read replica is being hammered for the next week, trying to get various proficiency metrics for the upcoming board meeting. If we are certain the migration will take less than twelve hours, we can work together to run it overnight. Otherwise, until I have all the numbers, I'm unwilling to give up a lock on the UserLevel table, even over this upcoming weekend (though hopefully I'll have all the numbers I need by then).